interfaces/builtin/browser_support_test.go

```diff
@@ -105,10 +112,15 @@ func (s *BrowserSupportInterfaceSuite) TestConnectedPlugSnippetWithoutAttrib(c *
 	c.Assert(string(snippet), Not(testutil.Contains), `capability sys_admin,`)
 	c.Assert(string(snippet), testutil.Contains, `deny ptrace (trace) peer=snap.@{SNAP_NAME}.**`)
 
-	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecuritySecComp)
+	seccompSpec := &seccomp.Specification{}
+	err = seccompSpec.AddConnectedPlug(s.iface, s.plug, s.slot)
 	c.Assert(err, IsNil)
-	c.Assert(string(snippet), testutil.Contains, `# Description: Can access various APIs needed by modern browers`)
-	c.Assert(string(snippet), Not(testutil.Contains), `chroot`)
+	snippets := seccompSpec.Snippets()
```

Comment:
I see this and the following 6 lines repeated a bunch of times - worthwhile to extract into a helper? The only difference seems to be if "chroot" is included or not.